### PR TITLE
Fix several tutorials on Windows

### PR DESCRIPTION
**Pull request by bellenot**
- Copy the missing `w32pragma.h` header in the `include` directory
- Change `G__WIN32` into `R__WIN32` in a couple of macros
- Don't compile the `timeonaxis3.C` macro (not needed anymore)

### CODE REVIEW

File: tutorials/eve/pythia_display.C [hunk 1]
```diff
@@ -16,7 +16,7 @@ void pythia_display()
    dir.ReplaceAll("/./","/");
    gROOT->LoadMacro(dir +"MultiView.C+");
 
-#ifndef G__WIN32 // libPythia6 is a static library on Windoze
+#ifndef R__WIN32 // libPythia6 is a static library on Windoze
```
Comment (oshadura) [inline tutorials/eve/pythia_display.C:19]: Windoze? :)

Comment (bellenot) [reply tutorials/eve/pythia_display.C:19]: Absolutely! 😉

File: tutorials/eve/pythia_display.C [hunk 2]
```diff
@@ -228,7 +233,7 @@ void pythia_next_event()
    gTrackList->MakeTracks();
 
 
-   TEveElement* top = gEve->GetCurrentEvent();
+   TEveElement* top = (TEveElement *)gEve->GetCurrentEvent();
```
Comment (eguiraud) [inline tutorials/eve/pythia_display.C:236]: `static_cast` is safer than a `reinterpret_cast`, if possible:

```suggestion
   TEveElement* top = static_cast<TEveElement *>(gEve->GetCurrentEvent());
```

**Review by eguiraud — APPROVED** [review]
LGTM. possibly add a comment explaining who/what needs w32pragma.h to be copied in the build directory (besides being installed as it already is)

**Comment by bellenot** [comment]
> LGTM. possibly add a comment explaining who/what needs w32pragma.h to be copied in the build directory (besides being installed as it already is)

Well, I mostly work with ROOT from the build directory, and I sometimes need w32pragma.h when building several applications, so it's the main reason...
And thanks for the review!

**Pull request merged**